packages/web-react/src/components/TextField/stories/argTypes.ts

```diff
@@ -65,9 +65,11 @@ export default {
   },
   message: {
     control: {
-      type: 'text',
+      type: 'object',
```

Comment:
Are you sure this type works correctly for an array?

Reply:
it works correctly IMHO. But I am open to suggestions. I need it to accept both string and array and it works this way.